nn_surrogate_benchmark/surrogate_sobol.py

```diff
@@ -0,0 +1,209 @@
+import torch
+import torch.nn as nn
+import pytorch_lightning as pl
+from torch.optim import AdamW, Adam
+from typing import Literal
+import pandas as pd
+import numpy as np
+from torch.utils.data import TensorDataset, DataLoader
+from sklearn.preprocessing import (
+	StandardScaler,
+	MinMaxScaler,
+	RobustScaler,
+	MaxAbsScaler,
+)
+
+class Sobolev(pl.LightningModule):
```

Comment:
maybe it could inherit from MLP? this way it'll be easier to maintain

Reply:
I'm not well versed enough to have an opinion on this choice. For now pytorch lightning seems fine for me